main/factor.R

```diff
@@ -0,0 +1,45 @@
+# redirect output to a file
+sink("factor.out")
+
+library(readr)
+library(nFactors)
```

Comment:
how important is this package? it's not easily available to me. it's also not in the CSV below?

Reply:
I *think* I can work around these by just using the bazel binary, which should pull in the external packages from CRAN.

Reply:
hmmm. If it's about `readr`, is in all R files I submitted. I use it to read in csv. It's also in the `r_packages.csv`
If it's about `nFactors`, I'm adding it to the csv